pydatview/GUISelectionPanel.py

```diff
@@ -564,6 +594,8 @@ def __init__(self, parent, selPanel, mainframe):
         self.lbColumns.SetFont(getMonoFont(self))
         # Events
         self.lbColumns.Bind(wx.EVT_RIGHT_DOWN, self.OnColPopup)
+        self.lbColumns.Bind(wx.EVT_MOTION, self.OnColMotion)
```

Comment:
This event removes the possibility to select "multiple columns" by simply dragging the mouse (something that works in windows, if you are using windows). Multiple column selection still works with "Ctrl". I wonder if there might be a way to "release" or continue the  previous behavior, which I believe is stopped by the try/catch.

Reply:
Unfortunately I cannot reproduce this on Linux at the moment. I will try as soon as possible..
Anyway, I added event propagation, so it might works already.

Reply:
`event.Skip()` did the trick!

Reply:
Great :)